### PR TITLE
UI kerngegevens to master

### PR DESCRIPTION
**Pull request by ghost**
This is the big one which hopefully will end the chaos. In Oscar we trust.

### CODE REVIEW

File: app/styles/_shame.scss [hunk 1]
```diff
@@ -13,7 +13,17 @@
 .data-table-pagination-left {
   @extend .au-c-help-text;
 }
-
+.helptext {
```
Comment (lagartoverde) [inline app/styles/_shame.scss:16]: Please use dashes to be consistent help-text link-external, active-status...

File: app/templates/core-data/admin-unit/index.hbs [hunk 2]
```diff
@@ -56,44 +57,59 @@
...
+                  {{@model.nis.localId}}
+                </:content>
+              </Item>
+            {{/if}}
+            {{#if @model.sharepoiint}}
+              <Item>
+                <:label>Sharepoint referentie
+                </:label>
+                <:content>
+                  {{!-- <AuLinkExternal
```
Comment (lagartoverde) [inline app/templates/core-data/admin-unit/index.hbs:99]: Remove commented code if it's no longer needed

File: app/templates/sites/site/edit.hbs [hunk 3]
```diff
@@ -71,12 +74,12 @@
...
                       @alignment="inline"
-                      @onChange={{fn (mut this.model.site.isPrimarySite)}}
-                      @selected={{this.model.site.isPrimarySite}}
+                      @onChange={{fn (mut this.selectedPrimaryStatus)}}
+                      @selected={{this.currentIsPrimary}}
                       as |Group|
                     >
-                      <Group.Radio @value="Ja">Ja</Group.Radio>
-                      <Group.Radio @value="Nee">Nee</Group.Radio>
+                      <Group.Radio @value={{true}}>Ja&nbsp;</Group.Radio>
```
Comment (lagartoverde) [inline app/templates/sites/site/edit.hbs:81]: Why are the non breaking spaces needed?

File: app/controllers/core-data/admin-unit/index.js [hunk 4]
```diff
@@ -8,4 +8,7 @@ export default class CoreDataAdminUnitIndexController extends Controller {
   get isMunicipality() {
     return this.model.classification.id === CLASSIFICATION_CODE.MUNICIPALITY;
   }
+  get testDate() {
```
Comment (lagartoverde) [inline app/controllers/core-data/admin-unit/index.js:11]: This should be removed I think

File: app/styles/_shame.scss [hunk 5]
```diff
@@ -91,3 +101,7 @@ $ember-power-select-focus-outline: solid 2px rgb(16, 108, 200) !default;
     }
   }
 }
+
+#vote-star-filled {
```
Comment (lagartoverde) [inline app/styles/_shame.scss:105]: I would change it to something more specific, like primary-site-icon or something like that

**Review by lagartoverde — CHANGES_REQUESTED** [review]
Some minor comments

**Review by lagartoverde — APPROVED** [review]

File: app/templates/core-data/admin-unit/index.hbs (hunk 2 above)
Comment (DennisVanEecke) [reply app/templates/core-data/admin-unit/index.hbs]: This was intentional. We do not seem to have all the data present. OP shows a web link. I kept the old code in comment for this reason. But I'll remove it.

File: app/controllers/core-data/admin-unit/index.js (hunk 4 above)
Comment (DennisVanEecke) [reply app/controllers/core-data/admin-unit/index.js]: Yup. It's gone

File: app/styles/_shame.scss (hunk 5 above)
Comment (DennisVanEecke) [reply app/styles/_shame.scss]: Changed to primary-site-indicator

File: app/styles/_shame.scss (hunk 1 above)
Comment (DennisVanEecke) [reply app/styles/_shame.scss]: Changed the names to be more consistent

**Pull request merged**